db/migrate/20170927135007_migrate_dialog_field_associations_to_use_new_relationship.rb

```diff
@@ -0,0 +1,44 @@
+class MigrateDialogFieldAssociationsToUseNewRelationship < ActiveRecord::Migration[5.0]
+  class DialogFieldAssociation < ActiveRecord::Base
+    include ReservedMigrationMixin
```

Comment:
We are not using Reserved data columns so this should not be needed in any of the models.